.github/workflows/bridge.yml

```diff
@@ -0,0 +1,158 @@
+        if: steps.cmfgen-cache.outputs.cache-hit != 'true'
+
+      - name: Setup carsus environment
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+            miniforge-variant: Mambaforge
+            miniforge-version: latest
+            environment-file: carsus/carsus_env3.yml
+            activate-environment: carsus
+            use-mamba: true
+
+      - name: Install package
+        run: pip install -e carsus/
+
+      - name: Run notebooks
+        run: |
+          ${{ env.NBCONVERT_CMD }} carsus/docs/refdata_gen.ipynb
+        env:
+          CARSUS_REFDATA: ${{ github.workspace }}/carsus-refdata
+        
+      - name: upload atom data
+        uses: actions/upload-artifact@v2
+        with:
+          name: atom-data
+          path: carsus/docs/kurucz_cd23_chianti_He_cmfgen_H_Si_I-II.h5
+  
+  tardis-build:
+    needs: carsus-build
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          repository: tardis-sn/tardis
+          
+      - uses: actions/checkout@v2
+        with:
+          repository: atharva-2001/tardis-refdata
+          ref: raise_error_refdata_compare
+          path: tardis-refdata
+          lfs: true
+        
+      - name: Fetch lfs objects
+        run: |
+             cd tardis-refdata
+             git lfs checkout 
+
+      - name: Setup TARDIS environment
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+            miniforge-variant: Mambaforge
+            miniforge-version: latest
+            activate-environment: tardis
+            use-mamba: true
+
+      - uses: actions/cache@v2
+        with:
+          path: /usr/share/miniconda3/envs/tardis
+          key: conda-linux-64-${{ hashFiles('conda-linux-64.lock') }}-${{ env.CACHE_NUMBER }}
+        id: cache-conda
+
+      - name: Update environment
+        run: mamba update -n tardis --file conda-linux-64.lock
+        if: steps.cache-conda.outputs.cache-hit != 'true'
+
+      - name: Install bokeh
+        run: mamba install bokeh=2.2 --channel conda-forge --no-update-deps --yes
+
+      - name: Install package
+        run: pip install -e .
+        
+      - name: Download atom data
+        uses: actions/download-artifact@v2
+        with:
+          name: atom-data
+          path: carsus/docs/kurucz_cd23_chianti_He_cmfgen_H_Si_I-II.h5      
+      
+      - name: Replace atomic data in reference data repository with the generated atomic data
+        run: cp carsus/docs/kurucz_cd23_chianti_He_cmfgen_H_Si_I-II.h5/kurucz_cd23_chianti_He_cmfgen_H_Si_I-II.h5 tardis-refdata/atom_data/kurucz_cd23_chianti_H_He.h5
+      
+      - name: Create copy of reference data for comparision
+        run: cp tardis-refdata/unit_test_data.h5 tardis-refdata/unit_test_data_org.h5 
+
+      - name: Run tests and generate new reference data
+        run:  pytest tardis ${{ env.PYTEST_FLAGS }}
+
+      - name: Run notebook to test spectrum
```

Comment:
Shorter name

Reply:
Fixed in https://github.com/tardis-sn/carsus/pull/313/commits/42c2bb40df53f0fef1a78755f48aa1176876f73b